routes/api/v1/dashboards.js

```diff
@@ -0,0 +1,69 @@
+const Dashboard = require('../../../models/dashboards.js')
```

Comment:
```js
const Dashboard = require( '../../../models/dashboard.js' )
```
```js
router.get( '/', ( request, response, next ) => {

})
```
I'm cool with the final sets of parens on a line collapsing:
```js
  .then( dashboard => response.json( dashboard ))
```
Also, we can omit the `next` function if we're not using it in the block.